### DISCUSSION TITLE
Digital signature certificate

### DISCUSSION THREAD

**Discussion by showbit01** [MatthiasValvekens/pyHanko#305] [Q&A]
I have a doubt regarding linking of certificate signature to pdf.
I have made annotations in botton of each page of pdf file now I don't know how to link 1 signature certificate pfx file  to each annotation so that it can reflect on each page.
Please help.

**Comment by MatthiasValvekens** [comment]
This is a question that gets asked every so often in any space that discusses PDF signing. The short answer is that while this is not super hard to do technically, it's explicitly disallowed by the PDF standard so support in viewers is not consistent. Also, the legal implications are murky at best. As such, I'd prefer to keep this "feature" out of pyHanko since I don't want to be on the hook for maintaining it---questions of the form "hey, why does this feature not work in PDF viewer XYZ" are not fun to have to answer ;).

You're of course free to hack something together yourself; see #70 for a more thorough answer that has some pointers on where to get started.
